third_party/blink/public/BUILD.gn

```diff
@@ -469,6 +473,10 @@ source_set("blink_headers") {
     "//ui/base:base",
     "//v8:v8_headers",
   ]
+
+  if (is_cobalt && use_starboard_media) {
+    deps += [ "//starboard($starboard_toolchain)" ]
```

Comment:
Is there any Starboard functions used in blink? If not, this is unnecessary. Same for all other places.